src/test/FuzzerImpl.cpp

```diff
@@ -216,6 +214,39 @@ TransactionFuzzer::inject(stellar::XDRInputFileStream& in)
     }
 }
 
+void
+TransactionFuzzer::generateFuzzerInput(const std::string& filename)
+{
+    XDROutputFileStream out;
+    out.open(filename);
+    autocheck::generator<Operation> gen;
+    try
+    {
+        xdr::xvector<Operation> ops;
+        auto numops = autocheck::generator<uint>()(6);
+        for (int i = 0; i < numops; ++i)
+        {
+            Operation op = gen(14);
+            while (isBadInput(op))
+                op = gen(14);
+            ops.emplace_back(op);
+        }
+
+        out.writeOne(ops);
+    }
+    catch (xdr::xdr_bad_discriminant const&)
```

Comment:
Is it possible to change the above so `xdr_bad_discriminant` never happens?

Reply:
I actually do not believe this is triggered since these xdr's are generated by `autocheck`, which knows enough to generate valid structures.